packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+## 5.2.3
+* Bump Android dependency on com.google.android.gms:play-services-auth and remove the need for jetifier.
```

Comment:
Bumps

(See discussion of verb form in the new CHANGELOG style guide linked from the PR checklist)

Reply:
done